### DISCUSSION TITLE
ESLint error when importing from `test-utils`; Missing file extension `.mts`

### DISCUSSION THREAD

**Discussion by aleeenaa** [thebuilder/react-intersection-observer#696] [Q&A]
react-intersection-observer: 9.13.0
node v18.20.2

Getting this linting error in my project
![Screenshot 2024-08-06 at 15 18 30](https://github.com/user-attachments/assets/455692ef-2f9b-4db6-800e-7b45c6a1046e)

**Comment by thebuilder** [comment]
Does it work? What if you add the extension to the import? Seems like your eslint rule expects the extension for all your imports, so it should be added
